kpi/tests/api/v2/test_api_assets.py

```diff
@@ -495,6 +496,54 @@ def test_asset_version_id_and_content_hash(self):
         self.assertEqual(response.data['version__content_hash'],
                          self.asset.latest_version.content_hash)
 
+    def test_count_endpoint(self):
```

Comment:
This test is with the owner if I'm not wrong. Would you mind to add tests with AnonymousUser and regular user? 
For the regular user, the test could start without 'view_submissions' then retry with `view_submissions` to ensure it pass with proper permission.